content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment. There are four levels of threading supported by
+the \openshmem{} implementation.
+ 
+\begin{itemize}
+\item {\bf \CONST{SHMEM\_THREAD\_SINGLE}} The \openshmem{} program may not be multithreaded, 
+and all \openshmem{} interfaces are invoked by a single thread. 
+
+\item {\bf \CONST{SHMEM\_THREAD\_FUNNELED}}
+The \openshmem{} program may be multithreaded, however, the 
+program must ensure that only the main thread invokes the \openshmem{}
+interfaces, i.e, the thread that invokes either \FUNC{shmem\_init} or
+\FUNC{shmem\_init\_thread}.
+
+\item {\bf \CONST{SHMEM\_THREAD\_SERIALIZED}} 
+The \openshmem{} program may be multithreaded, however, the 
+program must ensure that only one thread invokes the \openshmem{}
+interfaces at any given instance, i.e., the \openshmem{} interfaces 
+are not invoked concurrently by multiple threads.
```

Comment:
Strike: "only one thread invokes the \openshmem{} interfaces at any given instance, i.e., "